Sources/KituraSampleRouter/RouterCreator.swift

```diff
@@ -34,6 +34,25 @@ enum SampleError: Swift.Error {
     case sampleError
 }
 
+let customParameterHandler: RouterHandler = { request, response, next in
+    let id = request.parameters["id"] ?? "unknown"
+    response.send("\(id)|").status(.OK)
+    next()
+}
+
+class CustomParameterMiddleware: RouterMiddleware {
+    func handle(request: RouterRequest, response: RouterResponse, next: @escaping () -> Void) {
+        do {
+            try customParameterHandler(request, response, next)
+        } catch {
+            Log.error("customParameterHandler returned error: \(error)")
+        }
+
+    }
+}
+
+let router = Router()
```

Comment:
Can you explain this line? Surely we had a router already? I don't quite follow...